src/components/foooter/Footer.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { FaInstagram, FaFacebook, FaTwitter } from 'react-icons/fa';
+
+function Footer() {
+  return (
+    <div className="footer bg-white h-auto flex flex-col sm:flex-col  justify-between px-8 py-2 pt-10 gap-5 sm:gap-2 max-w-full">
+      <div className="flex flex-col sm:flex-row sm:gap-20">
+        <div className="footer__links flex  justify-center sm:justify-start">
+          <a href="/about-us" className="mr-6">
+            About us
```

Comment:
Also, i think if you can add this ```About us, Help & contact, Policies``` text in the middle right above the copyright text i think it'd look much better

Reply:
I have take look on different footer of many website I found that they use that way.